source/agora/node/Registry.d

```diff
@@ -540,6 +540,70 @@ private struct TypedPayload
     /// UTXO
     public Hash utxo;
 
+    /***************************************************************************
+
+        Construct an instance of an `TypedPayload` from REST payload
+
+        Params:
+            type = Resource record type
+            payload = Payload data of API
+            utxo = UTXO of the validator (optional)
+
+    ***************************************************************************/
+
+    public this (TYPE type, RegistryPayload payload, Hash utxo = Hash.init) @safe
```

Comment:
This is not needed as compiler already generates this by default.

Reply:
Since I've added a ctor (from RR) compiler stops generating default

Reply:
Ah yes, my bad